src/pages/iou/IOUCurrencySelection.js

```diff
@@ -0,0 +1,219 @@
+    currencyList: {},
+    onCurrencyConfirm: null,
+};
+
+class IOUCurrencySelection extends Component {
+    constructor(props) {
+        super(props);
+
+        const {currencyOptions} = getCurrencyListForSections(this.props.currencyList,
+            '');
+
+        this.state = {
+            searchValue: '',
+            currencyData: currencyOptions,
+        };
+        this.renderItem = this.renderItem.bind(this);
+        this.getSections = this.getSections.bind(this);
+    }
+
+    componentDidMount() {
+        getCurrencyList();
+    }
+
+    /**
+     * Returns the sections needed for the OptionsSelector
+     *
+     * @param {Boolean} maxParticipantsReached
+     * @returns {Array}
+     */
+    getSections() {
+        const sections = [];
+
+        sections.push({
+            title: 'ALL CURRENCIES',
+            data: this.state.currencyData,
+            shouldShow: true,
+            indexOffset: 0,
+        });
+
+        return sections;
+    }
+
+    /**
+     * Returns the key used by the list
+     * @param {Object} option
+     * @return {String}
+     */
+    extractKey(option) {
+        return option.currencyCode;
+    }
+
+    /**
+     * Function which renders a row in the list
+     *
+     * @param {String} currencyCode
+     *
+     */
+    toggleOption(currencyCode) {
+        this.props.onCurrencySelected({
+            currencyCode,
+            currencySymbol: this.props.currencyList[currencyCode].symbol,
+        });
+    }
+
+    /**
+     * Function which renders a row in the list
+     *
+     * @param {Object} params
+     * @param {Object} params.item
+     *
+     * @return {Component}
+     */
+    renderItem({item, key}) {
+        return (
+            <OptionRow
+                key={key}
+                mode="compact"
+                option={item}
+                onSelectRow={() => this.toggleOption(item.currencyCode)}
+                isSelected={item.currencyCode === this.props.selectedCurrency.currencyCode}
+                showSelectedState
+                hideAdditionalOptionStates
+            />
+        );
+    }
+
+    /**
+     * Function which renders a section header component
+     *
+     * @param {Object} params
+     * @param {Object} params.section
+     * @param {String} params.section.title
+     *
+     * @return {Component}
+     */
+    renderSectionHeader({section: {title}}) {
+        return (
+            <View>
+                <Text style={[styles.p5, styles.textMicroBold, styles.colorHeading]}>
+                    {title}
+                </Text>
+            </View>
+        );
+    }
+
+    render() {
+        const sections = this.getSections();
+        return (
+            <ScreenWrapper>
+                {() => (
+                    <>
+                        <View style={[styles.flex1, styles.w100]}>
+                            <View style={[styles.ph5, styles.pv3]}>
```

Comment:
Since this entire area is already wrapped in 20px padding, let's remove the padding top from this view.